### PR TITLE
Add Identification & File data models, and EntityModel base model

### PR DESCRIPTION
**Pull request by jnik-aarhus**
This PR will add the Identification and File data models, as well as the EntityModel base model, which will serve as a basis for future API related data models.

Version is bumped from 0.1.0 -> 0.3.1, since I believe that adding a base model is more of an "update" rather than a full model.

### CODE REVIEW

File: datamodels/__init__.py [hunk 1]
```diff
@@ -1 +1,16 @@
-__version__ = "0.1.0"
+# -----------------------------------------------------------------------------
+# Version
+# -----------------------------------------------------------------------------
+__version__ = "0.3.1"
```
Comment (clausjuhl) [inline datamodels/__init__.py:4]: Why 0.3 and not 0.2? Did you miss a version-update in the previous PR?

**Comment by clausjuhl** [comment]
Ups. Merged as well. That is not the job of a reviewer or?

File: datamodels/__init__.py (hunk 1 above)
Comment (jnik-aarhus) [reply datamodels/__init__.py:4]: We started at 0.1.0 and not 0.0.0 :)

**Comment by jnik-aarhus** [comment]
You can review & merge, it's fine!

File: datamodels/__init__.py (hunk 1 above)
Comment (clausjuhl) [reply datamodels/__init__.py:4]: I know. But why the jump from 0.1 to 0.3?

Comment (jnik-aarhus) [reply datamodels/__init__.py:4]: Because we added two models -- Identification and File Model

**Pull request merged**